Content.Shared/GameObjects/Components/Observer/GhostWarpToNameRequestMessage.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.GameObjects.Components.Observer
+{
+    [Serializable, NetSerializable]
+    public class GhostWarpToNameRequestMessage : ComponentMessage
+    {
+        public GhostWarpToNameRequestMessage(string warpTarget = default)
```

Comment:
1. I suggest renaming `warpTarget` to `name`.
2. Is it still required to put `= default` here?

Reply:
Done and fixed in https://github.com/space-wizards/space-station-14/pull/3310/commits/141de1126f31e66cc5db5d954ab65aadc3fbf794